usdigitalresponse/usdr-gost
# feat: rename agency status fields (#734)

### Ticket #
https://github.com/usdigitalresponse/usdr-gost/issues/734

### Description
Following some UXR feedback, there need to be some updates on the "Agency Status" fields in both the Grants modal and `Grants` tab

### Screenshots / Demo Video
<img width="998" alt="Screenshot 2023-01-11 at 11 49 16 AM" src="https://user-images.githubusercontent.com/12984659/211903854-c50b5bc8-cb3b-4357-af5d-f6b2e08572c7.png">
<img width="1008" alt="Screenshot 2023-01-11 at 11 50 05 AM" src="https://user-images.githubusercontent.com/12984659/211903860-41a41f30-68f5-476f-b287-974f926ff0f6.png">

### Testing
1. Grants Modal
- Go to the `Browse Grants` tab
- Click one of the grants
- Select `Agency Status` in the `GrantDetails`
- You should expect to see the updated values in both Result **(now: Applied)** and Rejections **(now: Not Applying)** sections

2. My Grants
- Go to the `My Grants` tab
- You should expect to see the updated tabs

### Checklist
